### PR TITLE
Update to v0.7.0

### PR DESCRIPTION
**Pull request by marcoesters**
Required for `tensorboard 2.12`, which is part of the `tensorflow` ecosystem.

Jira ticket: [PKG-1586](https://anaconda.atlassian.net/browse/PKG-1586)

[PKG-1586]: https://anaconda.atlassian.net/browse/PKG-1586?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by sumit0190 — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,23 +1,21 @@
...
 
 # Windows is currently unsupported, so we build an universal dummy wheel instead for Windows
-# See https://github.com/tensorflow/tensorboard/blob/348af4c/tensorboard/data/server/pip_package/build.py#L78-L86
+# See https://github.com/tensorflow/tensorboard/blob/data-server-v0.7.0/tensorboard/data/server/pip_package/build.py#L39-L43
 build:
-  skip: True  # [py<36]
+  skip: True  # [py<37]
   script_env:
     - WHEELNAME={{ _name }}-{{ version }}-py3-none-any.whl  # [win]
   number: 0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:21]: `    - openssl                           # [not (win or osx or linux-64)]` should be fixed as:

`    - openssl                           # [not (win or osx or (linux and x86_64)]`

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,23 +1,21 @@
...
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  # data server versions are not tagged, so we have to use the ID of the relevant rev
-  url: https://github.com/tensorflow/tensorboard/archive/{{ git_rev }}.zip
-  sha256: 712996fea67cea2a2bcc07debe46bebcecbac7b862f42d760cc3b2ed31f79a5a
+  url: https://github.com/tensorflow/tensorboard/archive/refs/tags/data-server-v{{ version }}.tar.gz
+  sha256: 47f0b4edf434cedea351f60ad7ebe63b575cfd103fa19555b404e0e108c364cd
   patches:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: Would be better:
`  patches:  # [win]`

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:21]: Missing `setuptools` https://github.com/tensorflow/tensorboard/blob/f1cb31c86d871e0258250248ab9488575410e784/tensorboard/data/server/pip_package/setup.py#L16 and `wheel` (as we install from wheels) in `host`

Comment (marcoesters) [reply recipe/meta.yaml:21]: Fixed, thank you!

**Pull request merged**